api/types/events/events.proto

```diff
@@ -1779,3 +1793,92 @@ message SessionUpload {
     // URL is where the url the session event data upload is at
     string SessionURL = 5 [ (gogoproto.jsontag) = "url" ];
 }
+
+// Identity matches github.com/gravitational/teleport/lib/tlsca.Identity except
+// for RouteToApp and RouteToDatabase which are nullable and Traits which is
+// represented as a google.protobuf.Struct (still containing a map from string
+// to strings).
+message Identity {
+    // Username is a username or name of the node connection
+    string Username = 1 [ (gogoproto.jsontag) = "user,omitempty" ];
+    // Impersonator is a username of a user impersonating this user
+    string Impersonator = 2 [ (gogoproto.jsontag) = "impersonator,omitempty" ];
+    // Groups is a list of groups (Teleport roles) encoded in the identity
+    repeated string Groups = 3 [ (gogoproto.jsontag) = "roles,omitempty" ];
```

Comment:
Could we have the field name and json tag match?

Reply:
I've used json tags that are already in use in other events for the same data; should I change the field name and break symmetry with `tlsca.Identity`?

Reply:
Right. I'm tempted to break symmetry, I'm not sure it is as important as having a clear proto API. Your call.

Reply:
Changed the field names 👍